microservices/frame_engine/config.py

```diff
@@ -0,0 +1,16 @@
+import os
+from pathlib import Path
+
+# input shape
+W = 640
+H = 640
+FPS = 120
+
+urlapi = 'http://localhost:8080/detect'
+
+CORAL_DATA_DIR = Path(__file__).parent / "../Coral_Data/"
+
+if not os.path.exists(CORAL_DATA_DIR):
+    os.makedirs(CORAL_DATA_DIR)
+
+data_test_directory = Path(__file__).parent / "../object_detection/google_coral/images/"
```

Comment:
This is a detail, but you can see here in github that this file doesn't have a newline, and there are a few reasons to have it, eg:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
https://www.reddit.com/r/Python/comments/1zjugg/question_why_does_pep8_recommend_leaving_a_blank/

After I finish writing up I'll try to configure a lint integration with this repository so these things can be sorted out automatically :) I don't want to bring many rules in this side, only this more simple and direct ones, I'll discuss those with you guys

Reply:
solved